File: Java/Algorithm/DrawSquare.java
class Main {
	public static void main (String[] args) {
		int altura = 5;
		int largura = 5;
		for (int j = 0; j < largura; j++) System.out.print("* ");
        System.out.print("\n");
        for (int i = 2; i < altura; i++) {
    		System.out.print("* ");
	        for (int j = 2; j < largura; j++) System.out.print("  ");
	        System.out.print("* \n");
	    } 
		for (int j = 0; j < largura; j++) System.out.print("* ");
	}
}

//https://pt.stackoverflow.com/q/420189/101

File: C/Algorithm/DrawSquare.c
#include<stdio.h>

int main(void) {
    printf("TELL ME THE NUMBER YOU WANT: \n");
    int n;
    scanf("%d", &n);
    for (int row = 0; row < n; row++) {
    	for (int column = 0; column < n; column++) printf("*");
    	printf("\n");
    }
}

//https://pt.stackoverflow.com/q/171183/101
